### PR TITLE
rfetch: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: rfetch.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# rfetch
+
+> A fast, pretty, and configurable fetch program to output system information.
```
Comment (navarroaxel) [inline rfetch.md:3]: We try to get an objective description here.
```suggestion
> A configurable fetch program to output system information.
```

**Review by einverne — APPROVED** [review]
Looks good after @navarroaxel 's suggestion.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution

**Review by navarroaxel — APPROVED** [review]

**Comment by kbdharun** [comment]
> @kbdharun It looks like I accidentally put the file in the root of the repo. Can you move it to  **pages/common**?

Sure, Thanks for the information.

**Pull request merged**